rakelib/utils.rake

```diff
@@ -0,0 +1,58 @@
+# run a command, pipe output through 'xcpretty' and store the output in CI artifacts
+def xcpretty(cmd, task, subtask = '')
+  name = (task.name + (subtask.empty? ? '' : "_#{subtask}")).gsub(/[:-]/, "_")
+  command = [*cmd].join(' && ')
+  xcpretty = `which xcpretty`
```

Comment:
Same remarks as in https://github.com/SwiftGen/SwiftGen/pull/269#pullrequestreview-24136495